README.md

```diff
@@ -48,21 +41,26 @@ cd build
 cmake .. -DCMAKE_BUILD_TYPE=Release
 make
 
-# Testy
+# Tests
 GTEST_COLOR=1 make test CTEST_OUTPUT_ON_FAILURE=TRUE
 ```
 
-Pliki wyjściowe:
+Output files:
+
+Library: build/libecho/*.so*
```

Comment:
This renders as italic .so. We want literal `*.so`. Also it would be nice if paths were formatted as inline code, it looks better.
```suggestion
Library: `build/libecho/*.so`
```